src/patterns/patterns.py

```diff
@@ -802,7 +802,7 @@ def get_busfactor_data(self, locc_metric='change-size-cos', metric='mul-changes-
                 authors_commits_df.iat[ind, authors_commits_df.columns.get_loc('dev_knowledge')] = d_commits/tot_commits # calculating dev_knowledge on the whole project/directory
             
             authors_commits_df.sort_values(by=['dev_knowledge'], ascending=False, inplace=True)
-            results = authors_commits_df
+            results = aggregated_df
```

Comment:
I don't understand this change. Can you explain a bit more either in code or commit message?

Reply:
I have also changed to calculating bus factor from file to folder level. Earlier I was looking at all the files in a folder but now I am aggregating like we discussed earlier. I already had that calculated in the aggregated_df, using it now.

Reply:
Still a bit confused. This code does not know seem to be needed. Am I wrong? I don't see authors_commits_df used anywhere.

            authors_commits_df["dev_knowledge"] = 0
            tot_commits = authors_commits_df[locc_metric].sum() # total number of commits on the project/directory
            for ind in authors_commits_df.index:
                d_commits = authors_commits_df[locc_metric][ind]
                authors_commits_df.iat[ind, authors_commits_df.columns.get_loc('dev_knowledge')] = d_commits/tot_commits # calculating dev_knowledge on the whole project/directory
            
            authors_commits_df.sort_values(by=['dev_knowledge'], ascending=False, inplace=True)

Reply:
Yes, that is correct. I was using it earlier but not anymore. Will remove this code.